util.js

```diff
@@ -0,0 +1,10 @@
+
+export function getDistinctValues(values, selector) {
+	return values.reduce((found, next) => {
+		let value = selector != null ? selector(next) : next;
+		if (!found.includes(value)) {
+			found.push(value);
+		}
+		return found
+	}, []);
+}
```

Comment:
I am not entirely sure what this method does.
Is it supposed to take in an array of inputs and produce a modified array only containing unique values?